toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -341,22 +342,22 @@ func printSummary(failedSRPMs, failedSRPMsTests map[string]*BuildResult, prebuil
 	logger.Log.Info("--------- Summary ---------")
 	logger.Log.Info("---------------------------")
 
-	logger.Log.Infof(color.GreenString("Number of prebuilt SRPMs:%12d", len(prebuiltSRPMs)))
-	logger.Log.Infof(color.GreenString("Number of prebuilt delta SRPMs:%6d", len(prebuiltDeltaSRPMs)))
-	logger.Log.Infof(color.GreenString("Number of skipped SRPMs tests:%7d", len(skippedSRPMsTests)))
-	logger.Log.Infof(color.GreenString("Number of built SRPMs:%15d", len(builtSRPMs)))
-	logger.Log.Infof(color.GreenString("Number of tested SRPMs:%14d", len(testedSRPMs)))
-	printErrorInfoByCondition(len(unresolvedDependencies)>0, "%s%3d", "Number of unresolved dependencies:", len(unresolvedDependencies))
-	printErrorInfoByCondition(len(blockedSRPMs)>0, "%s%13d", "Number of blocked SRPMs:", len(blockedSRPMs))
-	printErrorInfoByCondition(len(blockedSRPMsTests)>0, "%s%7d", "Number of blocked SRPMs tests:", len(blockedSRPMsTests))
-	printErrorInfoByCondition(len(failedSRPMs)>0, "%s%14d", "Number of failed SRPMs:", len(failedSRPMs))
-	printErrorInfoByCondition(len(failedSRPMsTests)>0, "%s%8d", "Number of failed SRPMs tests:", len(failedSRPMsTests))
+	logger.Log.Infof(color.GreenString(summaryLine("Number of prebuilt SRPMs:", len(prebuiltSRPMs))))
+	logger.Log.Infof(color.GreenString(summaryLine("Number of prebuilt delta SRPMs:", len(prebuiltDeltaSRPMs))))
+	logger.Log.Infof(color.GreenString(summaryLine("Number of skipped SRPMs tests:", len(skippedSRPMsTests))))
+	logger.Log.Infof(color.GreenString(summaryLine("Number of built SRPMs:", len(builtSRPMs))))
+	logger.Log.Infof(color.GreenString(summaryLine("Number of tested SRPMs:", len(testedSRPMs))))
+	printErrorInfoByCondition(len(unresolvedDependencies)>0, "%s", summaryLine("Number of unresolved dependencies:", len(unresolvedDependencies)))
+	printErrorInfoByCondition(len(blockedSRPMs)>0, "%s", summaryLine("Number of blocked SRPMs:", len(blockedSRPMs)))
+	printErrorInfoByCondition(len(blockedSRPMsTests)>0, "%s", summaryLine("Number of blocked SRPMs tests:", len(blockedSRPMsTests)))
+	printErrorInfoByCondition(len(failedSRPMs)>0, "%s", summaryLine("Number of failed SRPMs:", len(failedSRPMs)))
+	printErrorInfoByCondition(len(failedSRPMsTests)>0, "%s", summaryLine("Number of failed SRPMs tests:", len(failedSRPMsTests)))
```

Comment:
Format string argument `"%s"` should no longer be needed. See how `fmt.Printf` is written for instance, in case things break.